templates/tab/ts/default/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "teamsfx-template-tab",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Do we really need to update the version? I think we just generate the sample tab app for user and this is the version of user's tab app, not the version of our template.

Reply:
Fixed. Thx!